radar/2022-03-28/semanticore.md

```diff
@@ -0,0 +1,15 @@
+---
+title:      "Semanticore"
+ring:       trial
+quadrant:   tools
+---
+
+[Semanticore](https://github.com/AOEpeople/semanticore) is an open source tool fulfilling our needs for managing Changelogs and semantic releases.
```

Comment:
I guess `changelog` should be lowercase throughout the text, shouldn't it?
In line 8 it's lowercase and in line 9 again uppercase.